### PR TITLE
Clean-up post extraction from Kleros V2, support for Gnosis chain

### PR DESCRIPTION
**Pull request by adi274**
- Fixes the naming (Home -> Sender, Foreign -> Receiver)
- Rinkeby Testnet removal

### CODE REVIEW

File: contracts/hardhat.config.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ const config: HardhatUserConfig = {
       saveDeployments: true,
       tags: ["staging", "sender", "layer2"],
       companionNetworks: {
-        receiver: "goerli",
+        receiver: "chiado",
```
Comment (jaybuidl) [inline contracts/hardhat.config.ts:67]: Hmm we'll need to find a better solution, that won't scale. Here the ArbiGoerli->Gerli deployment is certainly affected.

File: contracts/src/FastBridgeReceiverOnEthereum.sol [hunk 2]
```diff
@@ -36,30 +36,30 @@ contract FastBridgeReceiverOnEthereum is IFastBridgeReceiver, ISafeBridgeReceive
     // *              Views                * //
     // ************************************* //
 
-    function isSentBySafeBridge() internal view override returns (bool) {
+    function isSentBySafeBridge() internal view virtual override returns (bool) {
```
Comment (jaybuidl) [inline contracts/src/FastBridgeReceiverOnEthereum.sol:39]: no need for virtual

File: contracts/hardhat.config.ts (hunk 1 above)
Comment (jaybuidl) [reply contracts/hardhat.config.ts:67]: Let's look for a better solution later in a separate branch.

File: contracts/src/FastBridgeReceiverOnEthereum.sol (hunk 2 above)
Comment (jaybuidl) [reply contracts/src/FastBridgeReceiverOnEthereum.sol:39]: Right it's for the mock, nvm.

**Review by jaybuidl — DISMISSED** [review]

File: contracts/hardhat.config.ts (hunk 1 above)
Comment (adi274) [reply contracts/hardhat.config.ts:67]: On it!

**Pull request merged**